source/virtualBuffers/__init__.py

```diff
@@ -109,14 +109,30 @@ def obj(self):
 
 	@property
 	def label(self):
-		if self.itemType == "landmark":

```

Comment:
The documentation for this property belongs in browseMode.TextInfoQuickNavItem.label, as that's where the base property is defined. Only because this touches a completely different file, I'd suggest we leave this for this PR.